decred/tests/integration/dcr/test_rpc.py

```diff
@@ -203,7 +203,8 @@ def readINI(path, keys):
     "0000000e590860091d85960f114d2e457d101825e82c901465e94aa768191dcae08475a842086ad954d6"
 )
 cFilterHeader414000 = ByteArray(
-    "412f12ed5bd92df6a8b17cf396697cd44d84ada67f5ca3b1c3f23f3b7619984b"
+    # "412f12ed5bd92df6a8b17cf396697cd44d84ada67f5ca3b1c3f23f3b7619984b"
+    "a854742e7b3b8cd4f48fd2568c021e24bde9aaf07f1898cf0a9c1f133bcfe115"
```

Comment:
I'm looking into this now. I'm assuming it has something to do with recent work on the header commitments, but I'm on 1.6.0-pre and I'm still seeing the `412f...` filter header.

Reply:
Based on off-github conversations, I think we should drop support for v1 filters and headers altogether. Since we're still pre-release-ish, I think we have some latitude to do that. You can just comment out the test for now and we can pick up the work in another PR or open an issue. 

That said, it's not clear why our results are different.  Might be worth saving a copy of your data folder for investigation.

Reply:
I undid my change to the test data (that I didn't mean to commit anyway, sorry).

As per Matrix conversation I rebuilt the CF indexes and I'm now getting the correct hash.